lib/cli/commands/config.js

```diff
@@ -28,51 +33,47 @@ configCommand.builder = function(cli) {
 			builder: noop,
 			middlewares: [baseMiddleware],
 		})
-		.example("$0 config set mavenSnapshotEndpointUrl http://example.com/snapshots/",
-			"Set a value for the mavenSnapshotEndpointUrl configuration")
-		.example("$0 config set mavenSnapshotEndpointUrl",
-			"Unset the current value of the mavenSnapshotEndpointUrl configuration");
+		.example("$0 config set ui5DataDir .ui5",
```

Comment:
Maybe we should make it clear that this is a path?

```suggestion
		.example("$0 config set ui5DataDir /path/to/.ui5",
```